newIDE/app/src/EventsFunctionsExtensionEditor/EventsFunctionConfigurationEditor/EventsFunctionPropertiesEditor.js

```diff
@@ -172,7 +164,6 @@ export default class EventsFunctionPropertiesEditor extends React.Component<
             {renderConfigurationHeader ? renderConfigurationHeader() : null}
             <ResponsiveLineStackLayout alignItems="center" noMargin>
               <Line alignItems="center" noMargin>
-                <img src="res/function32.png" alt="" style={styles.icon} />
```

Comment:
Good idea to remove this icon as it is not really useful now that the more specialized icon is present in the EventsFunctionsList. 

Nonetheless, I think that a user that select a different type here should see the icon in the EventsFunctionsList updated with the corresponding one.

I think you can use the props `onConfigurationUpdated` to force update the `EventsFunctionsList` used by the `EventsFunctionsExtensionEditor` (you would need to add a ref to the component)

Reply:
I took care of this 👍